src/components/_input.scss

```diff
@@ -2,33 +2,42 @@
 
 // --- SUI Input --- //
 // @include sui-input;
+// @include sui-input('large');
 
-@mixin sui-input {
+@mixin sui-input($size: 'medium') {
   @include input-placeholder {
     font-style: italic;
```

Comment:
Speaking about placeholder!
I consider quite restrictive to have by default an italic style. It would be great to have a custom sass var instead.
And, in my opinion it would be great if we have yet another default param for `placeholder color` to overwrite later.